phoebe/atmospheres/passbands.py

```diff
@@ -423,67 +409,97 @@ def save(self, archive, overwrite=True, update_timestamp=True, export_inorm_tabl
+                continue
+
+            if f'{atm.name}:Inorm' in self.content:
+                if export_to_pre25 and atm.name == 'blackbody':
+                    teffs = self.ndp['blackbody'].axes[0]
+                    log10ints = self.ndp['blackbody'].table['inorm@energy']['grid']
+                    bb_func_energy = interpolate.splrep(teffs, log10ints, s=0)
+                    log10ints = self.ndp['blackbody'].table['inorm@photon']['grid']
+                    bb_func_photon = interpolate.splrep(teffs, log10ints, s=0)
+
+                    bb_func = Table({'teff': bb_func_energy[0], 'logi_e': bb_func_energy[1], 'logi_p': bb_func_photon[1]}, meta={'extname': 'BB_FUNC'})
+                    data.append(fits.table_to_hdu(bb_func))
+                else:
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['inorm@energy']['grid'], name=f'{atm.prefix.upper()}NEGRID'))
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['inorm@photon']['grid'], name=f'{atm.prefix.upper()}NPGRID'))
+
+                content.append(f'{atm.name}:Inorm')
+
+            if f'{atm.name}:Imu' in self.content:
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['imu@energy']['grid'], name=f'{atm.prefix.upper()}FEGRID'))
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['imu@photon']['grid'], name=f'{atm.prefix.upper()}FPGRID'))
+                content.append(f'{atm.name}:Imu')
+
+                if export_to_pre25 and f'{atm.name}:Inorm' not in content:
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['imu@energy']['grid'][..., -1, :], name=f'{atm.prefix.upper()}NEGRID'))
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['imu@photon']['grid'][..., -1, :], name=f'{atm.prefix.upper()}NPGRID'))
+                    content.append(f'{atm.name}:Inorm')
+
+            if f'{atm.name}:ld' in self.content:
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['ld@energy']['grid'], name=f'{atm.prefix.upper()}LEGRID'))
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['ld@photon']['grid'], name=f'{atm.prefix.upper()}LPGRID'))
+                content.append(f'{atm.name}:ld')
+
+            if f'{atm.name}:ldint' in self.content:
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['ldint@energy']['grid'], name=f'{atm.prefix.upper()}IEGRID'))
+                data.append(fits.ImageHDU(self.ndp[atm.name].table['ldint@photon']['grid'], name=f'{atm.prefix.upper()}IPGRID'))
+                content.append(f'{atm.name}:ldint')
+
+            if f'{atm.name}:ext' in self.content:
+                if export_to_pre25 and atm.name == 'blackbody':
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['ext@energy']['grid'], name=f'{atm.prefix.upper()}EGRID'))
+                    data.append(fits.ImageHDU(self.ndp[atm.name].table['ext@photon']['grid'], name=f'{atm.prefix.upper()}PGRID'))
```

Comment:
Not sure I understand this. Aren't blackbody extinction grids also called BBXEGRID and BBXPGRID?

Reply:
they are now, but not in the pre-2.4.17 passbands.